sway/tree/container.c

```diff
@@ -799,6 +802,7 @@ void container_set_resizing(struct sway_container *con, bool resizing) {
 		return;
 	}
 
+	container_update_title_textures(con);
```

Comment:
This seems out-of-place. See the function description. Why is this necessary?

Reply:
So, it seems like this is used to re-render the textures at the end of an interactive resize. Other than the issue mentioned above:

- This doesn't work when doing a non-interactive resize.
- This doesn't update the texture while resizing.